src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/WindowsForms/MyAppFileAccessor.cs

```diff
@@ -27,12 +29,28 @@ internal class MyAppFileAccessor : IMyAppFileAccessor, IDisposable
     private const string SplashScreenProperty = "SplashScreen";
     private const string MinimumSplashScreenDisplayTimeProperty = "MinimumSplashScreenDisplayTime";
 
+    private const string DefaultMyappFileContents = "<?xml version=\"1.0\" encoding=\"utf-8\"?>" +
+                                                    "<MyApplicationData xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">" +
+                                                    "  <MySubMain>true</MySubMain>" +
+                                                    "  <MainForm>Form1</MainForm>" +
+                                                    "  <SingleInstance>false</SingleInstance>" +
+                                                    "  <ShutdownMode>0</ShutdownMode>" +
+                                                    "  <EnableVisualStyles>true</EnableVisualStyles>" +
+                                                    "  <AuthenticationMode>0</AuthenticationMode>" +
+                                                    "  <SaveMySettingsOnExit>true</SaveMySettingsOnExit>" +
+                                                    "</MyApplicationData>";
+    
     [ImportingConstructor]
-    public MyAppFileAccessor([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider, UnconfiguredProject project, IProjectThreadingService threadingService)
+    public MyAppFileAccessor([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider,
+        UnconfiguredProject project,
+        IProjectThreadingService threadingService,
+        IPhysicalProjectTree projectTree)
     {
         _serviceProvider = serviceProvider;
         _project = project;
         _threadingService = threadingService;
+        _storage = projectTree.TreeStorage;
+        _absolutePath = _project.MakeRooted("My Project\\Application.myapp");
```

Comment:
nit: Use string literal
```suggestion
        _absolutePath = _project.MakeRooted(@"My Project\Application.myapp");
```

Reply:
Done, thanks!